plexmodule.py

```diff
@@ -1,242 +1,235 @@
-import collections
-import configparser
-import coloredlogs
-import json
-import logging
-import os
-import re
-import requests
-import sys
-from guessit import guessit
-from plexapi.myplex import MyPlexAccount
-from plexapi.server import PlexServer
-
-logger = logging.getLogger('PlexAniSync')
```

Comment:
Removing the separate `logger` config is a bad idea if you ask me. That allows for configurability. I would suggest a change like this instead:
```python
logger = logging.getLogger(__name__)
```

Reply:
The reason for removing the logger is because I am using custom class of logger which still function the same but it's in the misc folder called log have a look around there.

Reply:
I missed that part I guess. I thought they were all logging to the root logger which makes filtering hard (i.e. `logging.getLogger('some.app').setLevel(logging.INFO)` :)

Reply:
Hahaha fair, I was working on it like around 3am ish probably why I changed integer to insert I realised after changing it they it was just a parameter so it wouldn’t have affected the code in anyway 😂 ohh well... I don’t think there will be any use for logger as of now since the custom log class it’s much better at filtering bloatware logs. Looking at Urllib 👀.